aptos-move/aptos-vm/src/natives.rs

```diff
@@ -44,8 +45,15 @@ pub fn aptos_natives(
             .filter(|(addr, module_name, _, _)| {
                 !(*addr == CORE_CODE_ADDRESS && module_name.as_str() == "event")
             }),
-        )
-        .collect()
+        );
+
```

Comment:
@vgao1996 is there a better, Rustier way of doing this conditional chaining?

Reply:
Try `Option::into_iter`?